### PR TITLE
replay: JS CDN docs

### PR DESCRIPTION
**Pull request by bruno-garcia**
image

See: https://github.com/getsentry/sentry-docs/issues/5894
See: https://github.com/getsentry/sentry-docs/issues/5903

### CODE REVIEW

**Comment by billyvg** [comment]
@bruno-garcia what's this 
image

File: src/platforms/javascript/common/install/cdn.mdx [hunk 1]
```diff
@@ -56,6 +56,24 @@ Sentry.init({
...
+  integrity="sha384-{{ packages.checksum('sentry.javascript.browser', 'replay.min.js', 'sha384-base64') }}"
+  crossorigin="anonymous"
+></script>
+```
+
+<Note>
+
+This is an addition to the bundles mentioned above for errors and tracing. 
+
+</Note>
```
Comment (imatwawana) [inline src/platforms/javascript/common/install/cdn.mdx:73]: I don't understand what we're saying with this note. "Additional" just communicates this is another available bundle, which seems like it's obvious. Do you want to say that it's new?

File: src/platforms/javascript/common/install/cdn.mdx [hunk 2]
```diff
@@ -56,6 +56,24 @@ Sentry.init({
...
+
+Please refer to <PlatformLink to="/session-replay/">the Session Replay documentation</PlatformLink> for more information.
```
Comment (imatwawana) [inline src/platforms/javascript/common/install/cdn.mdx:75]:
```suggestion
Please refer to our <PlatformLink to="/session-replay/">Session Replay documentation</PlatformLink> to learn more.
```

**Comment by bruno-garcia** [comment]
> @bruno-garcia what's this
> image

Interestingly that was my review to the original PR (turns out it never went through):

image


Somehow it renders correctly on the live docs. And the [snippet is the same](https://docs.sentry.io/platforms/javascript/session-replay/#installation):

image

File: src/platforms/javascript/common/install/cdn.mdx (hunk 1 above)
Comment (bruno-garcia) [reply src/platforms/javascript/common/install/cdn.mdx:73]: @imatwawana on the bundle above it's not an addition but rather a replacement. I was trying to emphasize that this one is not to be a replacement to `bundle.js` or `tracing.bundle.js`. You need either one of those *plus* this one. 
Any other way to suggest that? Or should I leave this out?

Comment (imatwawana) [reply src/platforms/javascript/common/install/cdn.mdx:73]: Ah okay. I would actually suggest putting this in plain text:
```suggestion
Use this bundle **in addition to** one of the bundles (for errors and tracing) above:
```

**Comment by bruno-garcia** [comment]
I'm missing the disclaimer that Replay is in Beta.

The CDN is already documented under Session Replay top level menu.
Moving this PR to draft right now, lets add this entry once Replay is out of beta.

**Review by billyvg — APPROVED** [review]

**Comment by bruno-garcia** [comment]
Lets merge this once Replay is in open beta, so anyone reading this can actually take advantage of it

**Comment by bruno-garcia** [comment]
Closing it in favour of @mydea's https://github.com/getsentry/sentry-docs/pull/6088

**Pull request closed without merging**